source/frontend/src/features/mapSideBar/disposition/models/DispositionSaleFormModel.ts

```diff
@@ -38,6 +52,20 @@ export class DispositionSaleFormModel {
 
     model.netProceedsBeforeSppAmount = calculateNetProceedsBeforeSppAmount(entity);
     model.netProceedsAfterSppAmount = calculateNetProceedsAfterSppAmount(entity);
+
+    model.dispositionPurchasers = entity.dispositionPurchasers?.map(
```

Comment:
not sure why this is not barking, but the result of this could be an undefined, that does not match the type in the model

Reply:
Updated